presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergUtil.java

```diff
@@ -509,6 +509,18 @@ public static Optional<String> tryGetLocation(Table table)
         }
     }
 
+    public static List<SortField> getSortFields(Table table)
+    {
+        try {
+            return table.sortOrder().fields().stream()
+                    .map(SortField::fromIceberg)
+                    .collect(toImmutableList());
+        }
+        catch (Exception e) {
+            log.warn(String.format("Unable to fetch sort fields for table %s: %s", table.name(), e.getMessage()));
+            return ImmutableList.of();
+        }
+    }
```

Comment:
Is it appropriate to directly ignore the sort fields' transform? As mentioned above, the iceberg table may be an external created table and registered into presto.